libhfcommon/util.c

```diff
@@ -154,7 +154,7 @@ bool util_PinThreadToCPUs(uint32_t threadno, uint32_t cpucnt) {
     }
 
     for (uint32_t i = 0; i < cpucnt; i++) {
-        if (pset_assign(set, i, NULL) != 0) {
+        if (pset_assign(set, (start_cpu + i), NULL) != 0) {
             PLOG_W("pset_assign(%" PRIu32 "), failed", i);
```

Comment:
I suggest adding ```% num_cpus``` too. Or, maybe it's solaris' default behavior?

Reply:
no you re right :)